components/Navigation/components/NavItem/NavItem.tsx

```diff
@@ -23,9 +24,9 @@ export const NavItem: FC<INavItemProps> = ({ active, icon, badge, ...props }) =>
 
   return (
     <div {...props} className={rootClassName} onMouseEnter={toggleHover} onMouseLeave={toggleHover}>
-      {badge ? <Badge color={active || isHovered ? 'outline-blue' : 'blue'}>{badge}</Badge> : null}
+      {badge ? <Badge color='outline-blue'>{badge}</Badge> : null}
```

Comment:
<img width="87" alt="Screenshot 2023-04-08 at 09 41 32" src="https://user-images.githubusercontent.com/55176773/230703495-5a6c1a29-b5fa-4739-875c-e7404f6c3179.png">

it should be like this
I think color='blue' will style it like so